pkg/agent/datamodel/const.go

```diff
@@ -130,6 +130,101 @@ const (
 	AKSGPUCudaSHA = "sha-ff213d"
 )
 
+/*
+	 If a new GPU sku becomes available, add a key to this map, but only if you have a confirmation
+		that we have an agreement with NVIDIA for this specific gpu.
+*/
+var NvidiaEnabledSKUs = map[string]bool{
+	// K80
+	"standard_nc6":   true,
+	"standard_nc12":  true,
+	"standard_nc24":  true,
+	"standard_nc24r": true,
+	// M60
+	"standard_nv6":      true,
+	"standard_nv12":     true,
+	"standard_nv12s_v3": true,
+	"standard_nv24":     true,
+	"standard_nv24s_v3": true,
+	"standard_nv24r":    true,
+	"standard_nv48s_v3": true,
```

Comment:
Thoughts on using apimachinery sets rather than maps here? Since all map entires are true probably makes more sense

Reply:
im fine either way, it would require clients to also take a dependency on apimachinery if they wanted to use this directly

Reply:
Either that or we could just use a map[string]struct{} so it uses zero bytes rather than taking api machinery dependency. There is a lot of useful deps in api machinery i would recommend adopting

Reply:
Decided to try out apimachinery sets, the only usages in RP where they are directly accessing the map is to get a string formatted list of the skus for logging. I created a helper function for this so that RP shouldn't have to add a dependency. I made all the map/sets private since they shouldnt need to be public now.